test/pricer_rule/utils.js

```diff
@@ -89,8 +97,10 @@ module.exports.createTokenEconomy = async (accountProvider) => {
     tokenRules.address,
   );
 
-  const quoteCurrencyCode = 'EUR';
-  const priceOracleInitialPrice = 11;
+  const quoteCurrencyCode = 'USD';
+  // $0.02 = 0.02*10^18(in contract)
```

Comment:
I would propose to remove this comment, as it's using `18` and also the below line is the same.

Reply:
Done!